packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -167,315 +173,383 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No calibration or manual settings",
+					"label": "No Calibration or Manual Settings",
 					"value": 0
 				},
 				{
-					"label": "Dimmer operates on auto-calibration settings",
+					"label": "Dimmer Operates on Auto-calibration",
 					"value": 1
 				}
 			]
 		},
 		"15": {
-			"label": "Burnt out bulb detection",
-			"description": "Percentage value of power variation",
+			"label": "Burnt Out Bulb Detection Power Variation (LOAD ERROR)",
+			"description": "Percentage value of standard power consumption as measured during calibration",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 99,
-			"defaultValue": 30
+			"defaultValue": 30,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		"16": {
-			"label": "Time delay of a burnt out bulb",
-			"description": "Time of delay for power variation detection",
+			"label": "Burnt Out Bulb Detection Time Delay (LOAD ERROR)",
 			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 5
+			"defaultValue": 5,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		"19": {
-			"label": "Forced switch on brightness level",
-			"description": "Dimmer 2 will always set this brightness level.",
+			"label": "Forced Switch On Brightness Level",
+			"description": "Percentage value.",
```

Comment:
```suggestion
```
Refer to the note here: https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=units
This is not a percentage ;)

Reply:
I read that, thus I put it as a description and not a unit :)

My reasoning here was that my interpretation of the Fibaro was that you can set a percentage between 1 and 99 and that will be used as the default brightness value, with 0% and 100% being unavailable. But I guess it could also be that the scale 1-99 is transformed to 0-100 or 1-100 percentages.

We also have parameter 1 and 2 that talks about brightness levels, and the value here is a different brightness. If parameter 1 and 2 specified the brightness level to be 10-50, we can still specify 75 on parameter 19 here and that will be a "real" brightness level of 40.

You don't think it is confusing if we have the same name when they mean different things?

Reply:
Or do I need to add something like?

`"description": "Relative to the range as set by minimum and maximum brightness level configuration"`